src/imperator/data/event_pictures.rs

```diff
@@ -29,14 +30,19 @@ impl DbKind for EventPicture {
         let mut sc = ScopeContext::new(Scopes::Country, key);
 
         vd.field_item("theme", Item::EventTheme);
-        vd.field_item("picture", Item::File);
 
-        vd.field_validated_block("picture", |block, data| {
-            let mut vd = Validator::new(block, data);
-            vd.field_item("texture", Item::File);
-            vd.field_validated_block("trigger", |b, data| {
-                validate_trigger(b, data, &mut sc, Tooltipped::No);
-            });
+        vd.multi_field_validated("picture", |bv, data| match bv {
+            BV::Value(t) => {
+                let mut vd = Validator::new(block, data);
+                vd.field_item("picture", Item::File);
```

Comment:
Here don't make another `vd`, because you have a value not a block.

You can do `data.verify_exists(Item::File, t)`